src/data/showcases.js

```diff
@@ -1066,6 +1066,15 @@ export const Showcases = [
     website: "https://nu.fi",
     source: null,
     tags: [ "wallet", "dex", "nftsupport"],
+  },  
+  {
+    title: "SecuX Hardware Wallet",
+    description:
+      "SecuX hardware wallets V20, W20 and W10 natively support the receiving, sending and staking of Cardano ADA on its web application Ada Portal. SecuX wallets are equipped with an Infineon SLE solid Flash CC EAL5+ Secure Element chip to securely store your PIN and Private Key, and authorize transactions without the private key ever leaving the device. Such vault-grade and military level security enables zero transaction leakage, and surpasses those adopted by banking systems as protection from malicious online attacks. The Ada Portal is accessed via the SecuXess web application, and connection verification will be required while switching from one to the other for extra security measures.",
+    preview: require("./showcase/SecuX.png"),
+    website: "https://secuxtech.com/",
+    source: null,
+    tags: [ "wallet", "Ecosystem"],
```

Comment:
```suggestion
    tags: [ "wallet"],
```
Two problems with this tag:
1) it doesn't exist (it's case sensitive; the proper spelling is `ecosystem`)
2) it's inappropriate, because "ecosystem" projects describe, inventory, or map the ecosystem: rather than just being parts of it.  See these projects already here & you will see what I mean: https://developers.cardano.org/showcase?tags=ecosystem

Reply:
Hi @rphair , the tag "ecosystem" has been removed, please help to confirm, thank you. 😊

Reply:
Thanks @NicoleYiyi for fixing the tags 🤓 although the other review issue remains unresolved (your `description:` field is 695 characters compared to our currently longest Description of 243 characters)